### PR TITLE
PR: Add pandoc as a dependency in our tests

### PR DESCRIPTION
**Pull request by rlaverde**
https://github.com/spyder-ide/spyder-reports/issues/27#issuecomment-317059351

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_description():
     return data
 
 
-REQUIREMENTS = ['spyder>=3.2.0.dev0', 'pweave']
+REQUIREMENTS = ['spyder>=3.2.0.dev0', 'pweave', 'pandoc']
```
Comment (ccordoba12) [inline setup.py]: Pandoc is not a Python package, so we can't add it to `setup.py`.

File: ci/install.sh [hunk 2]
```diff
@@ -15,11 +15,11 @@ conda install -q ciocheck -c spyder-ide --no-update-deps
 
 # Install dependencies
 if [ "$CIRCLE_NODE_INDEX" = "0" ]; then
-    pip install -q markdown pygments ipython nbformat nbconvert jupyter_client pyqt5 matplotlib
+    pip install -q pandoc markdown pygments ipython nbformat nbconvert jupyter_client pyqt5 matplotlib
     pip install git+ssh://git@github.com/mpastell/Pweave.git
 else
     conda install -q matplotlib
-    pip install -q pweave
+    pip install -q pweave pandoc
```
Comment (ccordoba12) [inline ci/install.sh]: Please remove it from here and add it to the line above, i.e.

    conda install -q matplotlib pandoc

File: ci/install.sh [hunk 3]
```diff
@@ -15,11 +15,11 @@ conda install -q ciocheck -c spyder-ide --no-update-deps
 
 # Install dependencies
 if [ "$CIRCLE_NODE_INDEX" = "0" ]; then
-    pip install -q markdown pygments ipython nbformat nbconvert jupyter_client pyqt5 matplotlib
+    pip install -q pandoc markdown pygments ipython nbformat nbconvert jupyter_client pyqt5 matplotlib
```
Comment (ccordoba12) [inline ci/install.sh]: This can't go here.

File: ci/install.sh (hunk 2 above)
Comment (ccordoba12) [inline ci/install.sh]: This is because conda can handle packages that are not only Python packages.

File: setup.py (hunk 1 above)
Comment (ccordoba12) [inline setup.py]: In reality, pandoc is Haskell package.

Comment (rlaverde) [inline setup.py]: Ohh sorry, I didn't know

**Pull request merged**